src/rules/no-descending-specificity/README.md

```diff
@@ -105,3 +105,9 @@ a {}
   #baz a {}
 }
 ```
+
+```css
```

Comment:
There's no need to add this to the README as it's an edge-case. We try to keep the READMEs as simple as possible and only show common patterns. Can you please remove this pattern.